retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc

```diff
@@ -438,7 +438,7 @@ Follow these steps to create the Apex Triggers:
 . Search for the Salesforce object. This opens a page that shows the details of the object.
 . Click *Triggers* in the Details section to view a list of available triggers for the object.
 . Click *New*.
-. Search for the Apex Trigger name, which is indicated in the previous table. Copy the Trigger Script code `CreateApexTriggers.txt` available in the `rcg-common-resources/salesforce` folder of https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources/[RCG Common Resources] project and paste it into the script editor.
+. Search for the Apex Trigger name, which is indicated in the previous table. Copy the Trigger Script code `CreateApexTriggers.txt` available in the `rcg-common-resources/salesforce` folder of https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project and paste it into the script editor.
```

Comment:
```suggestion
. Search for the Apex Trigger name, which is indicated in the previous table. Copy the Trigger Script code `CreateApexTriggers.txt` from the `rcg-common-resources/salesforce` folder of the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project and paste it into the script editor.
```

Reply:
This is one of those changes that needs to be back-ported to beta.